setup.py

```diff
@@ -7,7 +7,7 @@
 
 requirements = [
     # The core package.
-    "signac>=1.0.0",
+    "signac>=1.3.0",
```

Comment:
Is the assumption that this version is going to be fully compatibel with signac 2? Then, I'd suggest:
```suggestion
    "signac~=1.3,~=2.0",
```
Otherwise:
```suggestion
    "signac~=1.3",
```

Reply:
I currently expect it to be compatible with signac 2.0, yes. However, there are a few catches to discuss.

First, I think the proposed syntax `~=1.3,~=2.0` may be invalid. [PEP 440](https://www.python.org/dev/peps/pep-0440/#version-specifiers) says that "The comma (",") is equivalent to a logical **and** operator", which seems that it would break `~=1.3,~=2.0`. Based on the definition of a compatible release specifier, `~V.N` is like `>= V.N, == V.*`. No release could match both criteria `~=1.3` and `~=2.0` -- we would want an "or" separating them, if anything.

As an alternative, it might also be equivalent to specify `>=1.3.0,<3.0.0`. However, I recently read a perspective on why blocking a major version like this may pose a significant detriment to users: https://hynek.me/articles/semver-will-not-save-you/

> On the other hand, if a new major version of a package surprisingly breaks your package, they can always add a pin themselves (see step 4 above) until you fix your package. But there’s no practical way for them to remove your pin.
_Don’t ever pin major versions, unless you know they’re broken._ 

Since we won't truly know compatibility with signac 2.0 (or 3.0) until some later point in time when those releases are made, I recommend that we only bump the minimum version requirement for now, from `>=1.0.0` to `>=1.3.0`.

Reply:
You are absolutely, right, my suggestion with `~=1.3,~=2.0` is nonsensical, sorry about that.

I read the article you linked, certainly an interesting read although I don't fully agree with all of its conclusions. That said, I agree that updating our dependency to `>=1.3.0` is likely the best solution.

Reply:
Sounds good. I agree -- the article takes a strong stance on some topics, but I think there may be good reasons to pause before blocking major versions. 👍